integration_test/TezosMessageCodec.spec.ts

```diff
@@ -0,0 +1,61 @@
+// Most unit tests are commented out as they can only be run one by one with delays.
+// Uncomment specific unit tests to test specific operation logic.
+import { expect } from "chai";
+import { TezosOperations } from "../src";
+import { TezosMessageCodec } from "../src/tezos/TezosMessageCodec";
+import "mocha";
+import { servers } from "../test/servers";
+import { TezosWallet } from "../src";
+import { KeyStore } from "../src/types/KeyStore";
+
+const tezosURL = servers.tezosServer;
+
+//Software tezos operations do not require a valid derivation path
+const invalidDerivationPath = "ighiehgieh";
+
+// const keys = <KeyStore> TezosWallet.unlockFundraiserIdentity(
+//     'bomb sing vacant repair illegal category unveil color olive chest wink expand fringe pioneer reward',
+//     'efcvoykz.kygxsosz@tezos.example.org',
+//     'BFdu9bDJxC'
+// );
+
+describe("signOperationGroup() and computeOperationHash()", () => {
```

Comment:
Could we test one operation of each kind we currently can parse? Right now, we are testing only two.